docs/release-notes/job-queued-stats.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Bug Fixes**
+
+-  Fix an issue where job queued stats for jobs submitted to k8s would incorrectly show a very high
```

Comment:
I honestly would remove this release note since the feature still doesn't really work for Kubernetes and users still may have weird looking charts from allocations created on a previous version